cmd/podman/machine/console.go

```diff
@@ -0,0 +1,56 @@
+//go:build amd64 || arm64
+// +build amd64 arm64
+
+package machine
+
+import (
+	"fmt"
+
+	"github.com/containers/common/pkg/completion"
+	"github.com/containers/podman/v4/cmd/podman/registry"
+	"github.com/containers/podman/v4/pkg/machine"
+	"github.com/spf13/cobra"
+)
+
+var (
+	getConsoleCmd = &cobra.Command{
+		Use:               "get-console [NAME]",
```

Comment:
I would just name it `console`.

Reply:
For what it's worth, there is e.g. `aws ec2 get-console-output` to be clear that it just fetches.  Hmm.  Thinking about this a bit more if we were to make a non-SSH way to login, I think it'd be `podman machine login`.

(There's a whole sub-thread here of whether we should actually enable `podman machine exec`...)